packages/tasks/src/local-apps.ts

```diff
@@ -108,23 +108,18 @@ function getQuantTag(filepath?: string): string {
 const snippetLlamacpp = (model: ModelData, filepath?: string): LocalAppSnippet[] => {
 	const command = (binary: string) => {
 		const snippet = ["# Load and run the model:", `${binary} -hf ${model.id}${getQuantTag(filepath)}`];
```

Comment:
Temporary solution until we have the `--browser` flag

```suggestion
		const snippet = ["# Load and run the model:", `${binary} -hf ${model.id}${getQuantTag(filepath)}`, "# Then, access http://127.0.0.1:8080"];
```

Reply:
it think it's okay, since llama.cpp does show where to go in the CLI output. I'd prefer to keep it clean